newIDE/app/src/InstancesEditor/InstancesRenderer/LayerRenderer.js

```diff
@@ -248,7 +248,7 @@ export default class LayerRenderer {
   render() {
     this._computeViewBounds();
     this.instances.iterateOverInstancesWithZOrdering(
-      this.instancesRenderer,
+      gd.castObject(this.instancesRenderer, gd.InitialInstanceFunctor),
```

Comment:
The cast sadly won't work because they are not integrated with the typings.
You can just keep the original code and add a `// $FlowFixMe - gd.castObject is not supporting typings.` above the erroring line.